### ISSUE TITLE
 Can I use other datasets?

### ISSUE THREAD

**Issue by pacifinapacific** [Lotayou/densebody_pytorch#20]
Thank you for your hard work. Great job! I want to try your method, but I haven't got the human 3.6m dataset yet.
Can I try with other datasets like the SURREAL dataset or DenseposeCoCo?

**Comment by Lotayou** [comment]
Of course! The densebody paper actually tested their method on three benchmark datasets: Human36m, SURREAL and UP-3D, and it is exactly part of my next plan to add support for other datasets. I guess SURREAL would be a good starting point, and the UV map format exactly matches my current support list, just make sure you turn `uv_map` options in training scripts to `smpl_fbx`.

However for Densepose Coco you may want to be careful. As far as I know, densepose features a different 24-patch UV segmentation regime, which is not included in my current project. You may have to parse densepose UV map yourself for the concern of UV compatibility. You can checkout `UV_MAPS.md` for more info about using customized UV maps.

**Comment by pacifinapacific** [comment]
Thank you very much. I wait for support for the SURREAL data set to be added!

**Comment by pacifinapacific** [comment]
Thank you very much. Let's try a challenge with SURREAL dataset. And is it also possible to publish a trained parameter file?

**Comment by Lotayou** [comment]
I'm afraid I can't release pre-trained model due to copyright reasons. You can just train your own model following the instructions, should be straight-forward. Good luck:)

**Comment by wissalsayhi** [comment]
Your work is great, I would like to reproduce on a data set that I created by scanning people. my dataset has 200 different Human body obj file. I have the default uv maps. My main issue at this point is the annotation file. I don't have any idea how to reproduce it and fit my dataset to this code. Is it possible to do so?

**Comment by Lotayou** [comment]
@wissalsayhi I think it's quite possible, however I can use a little more info here. Are these human body files textured? What's the uv map looks like? Better yet, if you don't mind send me a sample of your scanned dataset, maybe I can work out a decent solution with it. My email is lingbo AT pku DOT edu DOT cn.

**Comment by dytan** [comment]
@Lotayou Thanks for your work. I have gotten the h36m dataset but unable to get the Moshed label, so tried with the toy example, and noticed these image name or index inconsistent with the raw h36m set, is there some determinated relation between the two image index order?  And may I get the Mosh annotation from you? The email is  dytan95 AT alumni.sjtu.edu.cn. Thank you.

**Comment by Lotayou** [comment]
@dytan Unfortunately no. I also found this toy dataset elsewhere and has no idea how these individual frames are sampled, as well as the generation of SMPL parameters. What a pity:)

**Comment by dytan** [comment]
@Lotayou  Thanks to know that.

**Comment by ananth-jauntxr** [comment]
Thanks for publishing the code :)

Any update on SURREAL dataset integration?

**Comment by Lotayou** [comment]
@ananth-jauntxr Sorry to let you wait for so long:) I indeed tried to develop an interface for reading SURREAL dataset before, but it turns out SURREAL has a very different annotation format and I don't know how to interpret it... I've temporarily moved on to other research projects but will come back to this once I tie up all loose ends, thanks again for your attention and patience.
